src/app/[locale]/map-center/page.tsx

```diff
@@ -1,14 +1,20 @@
+'use client';
+
 import dynamic from 'next/dynamic';
+import { useSearchParams } from 'next/navigation';
 
 const MapCenterPage = () => {
+  const searchParams = useSearchParams();
+  const entryType = searchParams.get('entryType'); // Get 'entryType' from URL query params
 
+  // Dynamically import the MapCenter component
   const MapCenter = dynamic(() => import('@/components/shared/map/MapCenter'), {
     ssr: false,
-  })
+  });
 
   return (
     <>
-      <MapCenter />
+      <MapCenter entryType={entryType as 'search' | 'sell'} /> {/* Pass entryType as a prop */}
```

Comment:
entryType appears to always be null.  Are you sure it is being retrieved correctly?